### PR TITLE
Generate manifest appache file

### PR DESCRIPTION
**Pull request by daaray**
Work on #7

### CODE REVIEW

**Comment by mlavin** [comment]
Failure on CI due to the compress directory not existing on a fresh build. Should be possible to work around.

**Comment by mlavin** [comment]
Once you get this test to pass :ship:

File: hygiene/tests/test_views.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        super(TempDirectoryMixin, self).tearDown()
+        # Remove temp dir
+        shutil.rmtree(self.temp_dir, ignore_errors=True)
+
+
+class CacheManifextTestCase(TempDirectoryMixin, TestCase):
+    def test_get_page(self):
+        """Audit detail"""
+        with self.assertTemplateUsed('hygiene/manifest.appcache'):
+            with self.settings(LOGIN_URL=self.temp_dir):
```
Comment (mlavin) [inline hygiene/tests/test_views.py]: `LOGIN_URL`?

Comment (daaray) [inline hygiene/tests/test_views.py]: oh god
fire me

**Comment by mlavin** [comment]
:rainbow: :rainbow:

**Pull request merged**